src/components/NewPriorityForm.js

```diff
@@ -56,16 +57,48 @@ export default class NewPriorityForm extends Component {
                     </label>
                     <input className="form__input" type="text" name="name" value={this.state.name} onChange={this.saveToState} placeholder="Textfield" required />
 
-                    <label className="form__label" htmlFor="details">
-                        Priority Details
-                    </label>
-                    <textarea className="form__textarea" type="text" name="details" value={this.state.details} onChange={this.saveToState} placeholder="Textfield" required />
-                    <button className="form__btn btn btn--primary-blue u-margin-top-small" type="submit">
-                        Add Priority
-                    </button>
-                </form>
-                </div>
-            </div>
-        );
-    }
+  createNewPriority = async e => {
+    e.preventDefault();
+    fetch(`${apiUrl}/priorities`, {
+      method: "POST",
+      mode: 'cors',
+      headers: { "Content-Type": "application/json" },
+      body: JSON.stringify({
+        priority_type_id: this.state.priority_type_id,
+        description: this.state.description,
+        visibility: true,  // hard coded for db
+        priority_status_type_id: 1,  // what exactly is this?
+        organization_id: this.state.organization_id,
+        rank: this.state.rank,
+        user_id: 9,  // hard coded for db
+      })
+    })
+      .then(res => console.log(res))
+      .catch(err => console.error(err))
+  }
+
+  render() {
+    return (
+      <div>
+        <Header title={"Add New Priority"} />
+        <div className="formpages">
+          <HeaderBlock name={"Add New Priority"} description={"Noticed something new in your community?"} />
+          <form className="form" name="new-priority" onSubmit={this.createNewPriority}>
+
+            <label className="form__label" htmlFor="priority_type_id">Priority Type</label>
+            <select name='priority_type_id' onChange={this.saveToState}>
+              {this.state.types.map(type => (
+                <option value={type.id} key={type.id}>{type.name}</option>
+              ))}
+            </select>
+
+            <label className="form__label" htmlFor="description">Priority Description</label>
+            <textarea className="form__textarea" type="text" name="description" value={this.state.description} onChange={this.saveToState} placeholder="Textfield" required />
```

Comment:
shouldn't the placeholder be 'description' ?

Reply:
Yea, I'll change that in my media query pr. Also, for this pr specifically, doesn't the master already contain these changes? Specifically lines: 61-68, 70-71, 73, 76, 80?

Reply:
The only thing in master that's the same (now) is the JSX return statement, but it needs to be updated anyway. It looks like it's changing way more because we aren't all using linters, or somehow using diffferent ones. @Kaleidics the create-react-app eslint setup defaults to 2 spaces, it looks like you might have either have another config overriding that or don't have eslint enabled on your editor so it's reformatting to 4 spaces?

Reply:
Yea, I'm using Prettier and I have it set to some custom settings so that may be why.

Reply:
@Kaleidics Or...it could be my prettier.  At some point, we need to figure that out to make our PR's easier to follow.